src/billable-services/payment-modes/add-payment-mode.modal.tsx

```diff
@@ -32,15 +38,19 @@ const AddPaymentModeModal: React.FC<AddPaymentModeModalProps> = ({ closeModal, o
   } = useForm<PaymentModeFormValues>({
     resolver: zodResolver(paymentModeSchema),
     defaultValues: {
-      name: '',
-      description: '',
+      name: editPaymentMode?.name,
+      description: editPaymentMode?.description,
```

Comment:
```suggestion
      name: editPaymentMode?.name ?? '',
      description: editPaymentMode?.description ?? '',
```

We could provide fallback values in case the values from the response are nullish or falsy.